apps/web/components/getting-started/steps-views/UserSettings.tsx

```diff
@@ -112,10 +117,17 @@ const UserSettings = (props: IUserSettingsProps) => {
       </div>
       <Button
         type="submit"
-        className="mt-8 flex w-full flex-row justify-center"
-        disabled={mutation.isPending}>
-        {t("next_step_text")}
-        <ArrowRight className="ml-2 h-4 w-4 self-center" aria-hidden="true" />
+        className="mt-8 flex w-full  flex-row justify-center "
+        disabled={mutation.isPending}
+        onClick={handleClick}>
+        {count ? (
+          <div className="cursor-wait px-[220px] py-2  "> Processing...</div>
+        ) : (
+          <>
+            {t("next_step_text")}
+            <ArrowRight className="ml-2 h-4 w-4 self-center" aria-hidden="true" />
+          </>
+        )}
```

Comment:
Please use the loading prop that is attached to the button component. 

This can just be loading={mutaiton.isPending} there is no need to implement state here